cli/gon-config.json

```diff
@@ -2,7 +2,7 @@
   "source": ["../target/production/wallet"],
   "bundle_id": "org.iota.cli-wallet",
   "apple_id": {
-    "password": "@env:AC_PASSWORD"
+    "provider": "UG77RJKZHH"
```

Comment:
How does that translate to our password?

Reply:
The default behavior is now to look for the value of the `AC_PASSWORD` env var if nothing is provided in the config file. the `provider` is a team ID on Apple's side